js-basics/simple-calculator/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -60,25 +66,31 @@ const multiplyByTwoButton = document.querySelector(
 const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
 
 increaseByOneButton.addEventListener("click", () => {
+  console.log(operand1 + 1);
```

Comment:
Here you don't change the variable, you just output the result of the expression to the console, if I press the button a few times, the variable will remain at 14, but the console will output 15 a few times. And we need the variable to increase by 1 each time I press the button.